utils.sh

```diff
@@ -297,7 +297,7 @@ function auto_conf_data () {
     amq_port=${AMQ_PORT}
     cws_amq_jmx_port=${CWS_AMQ_JMX_PORT}
     cws_jmx_port=${CWS_JMX_PORT}
-    history_days_to_live=1
+    history_days_to_live=30
```

Comment:
Does this get overrided from the properties file from the user?

Reply:
Yep - this gets overwritten by the configuration. I've gone ahead and changed it back to 1 (not a necessary change)